spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue79Test.java

```diff
@@ -21,7 +17,6 @@ class Issue79Test extends AbstractIntegrationTest {
     @Test
     void testIssue() {
         performAnalysis("ghIssues/Issue79.class");
-        BugInstanceMatcher bugMatcher = new BugInstanceMatcherBuilder().build();
-        assertThat(getBugCollection(), containsExactly(0, bugMatcher));
+        assertNoBugType("OBL_UNSATIFIED_OBLIGATION");
```

Comment:
Original issue (https://github.com/spotbugs/spotbugs/issues/79) is about OBL_UNSATIFIED_OBLIGATION FP.